docs/en/03_configuration.md

```diff
@@ -242,34 +254,41 @@ E.g.
 Or if you want to use local time in a summary field (where server time is UTC)
 
 ```php
- private static $summary_fields = [
-    'Title'                           => 'Title',
-    'Created'                         => 'Created (utc)',
-    'Created.getLocalTime.LocalValue' => 'Created (local)',
-];
+namespace App\Models;
+
+use SilverStripe\ORM\DataObject;
+
+class MyDataObject extends DataObject
+{
+    private static $summary_fields = [
+        'Title' => 'Title',
+        'Created' => 'Created (utc)',
+        'Created.getLocalTime.LocalValue' => 'Created (local)',
+    ];
+}
 ```
 
-## SilverStripe Fluent and search
+## Silverstripe `Fluent` and search
 
-### SilverStripe Core search
+### Silverstripe core search
 
-To use SilverStripe Core search with Fluent, add the `FluentSearchForm` to your configuration.
+To use Silverstripe Core search with Fluent, add the [`FluentSearchForm`](api:TractorCow\Fluent\Search\FluentSearchForm) to your configuration.
 
-```yaml
+```yml
 ---
 Name: myextensions
 ---
 SilverStripe\Core\Injector\Injector:
   SilverStripe\CMS\Search\SearchForm:
     class: TractorCow\Fluent\Search\FluentSearchForm
-``` 
+```
 
-**Note:** If you're using the SilverStripe CMS `$SearchForm` to display the default SilverStripe template on the front-end, this will update form and input id and class selectors from 'SearchForm' to 'FluentSearchForm'. E.g. form element `id='SearchForm_SearchForm'` will become `id='FluentSearchForm_SearchForm'`.
+**Note:** If you're using the Silverstripe CMS `$SearchForm` to display the default Silverstripe template on the front-end, this will update form and input id and class selectors from 'SearchForm' to 'FluentSearchForm'. E.g. form element `id='SearchForm_SearchForm'` will become `id='FluentSearchForm_SearchForm'`.
```

Comment:
```suggestion
> [!NOTE]
> If you're using the Silverstripe CMS `$SearchForm` to display the default template on the front-end, this will update form and input id and class selectors from 'SearchForm' to 'FluentSearchForm'. E.g. form element `id='SearchForm_SearchForm'` will become `id='FluentSearchForm_SearchForm'`.
```
It doesn't show up well in the diff but note I removed "Silverstripe" before "template"